sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/CommandHandlerProvider.java

```diff
@@ -38,8 +39,18 @@
     public Map<String, CommandHandler> namedHandlers() {
         Map<String, CommandHandler> map = new HashMap<String, CommandHandler>();
         List<CommandHandler> handlers = spiLoader.loadInstanceList();
+        List<CommandHandlerInterceptor> commandHandlerInterceptors = SpiLoader.of(CommandHandlerInterceptor.class).loadInstanceListSorted();
         for (CommandHandler handler : handlers) {
             String name = parseCommandName(handler);
+            if (!commandHandlerInterceptors.isEmpty()) {
+                List<CommandHandlerInterceptor> interceptors = new ArrayList<>();
+                for (CommandHandlerInterceptor commandHandlerInterceptor : commandHandlerInterceptors) {
+                    if (commandHandlerInterceptor.shouldIntercept(name)) {
+                        interceptors.add(commandHandlerInterceptor);
+                    }
+                }
+               handler = new InterceptingCommandHandler(handler, interceptors);
```

Comment:
if there is no interceptors mapping, I think we don't need the wrap the handler as `InterceptingCommandHandler`.

Reply:
> if there is no interceptors mapping, I think we don't need the wrap the handler as `InterceptingCommandHandler`.

I don't understand here, I have called ````if (!commandHandlerInterceptors.isEmpty())```` before, that will not wrap as  ````InterceptingCommandHandler````

Reply:
If no interceptors match, the handler is still wrapped as InterceptingCommandHandler.

Reply:
> If no interceptors match, the handler is still wrapped as InterceptingCommandHandler.
OK, I understand this time

Reply:
> If no interceptors match, the handler is still wrapped as InterceptingCommandHandler.

OK, I understand this time